build.gradle

```diff
@@ -123,7 +123,8 @@ subprojects {
     ]
 
     List junit = [
-            "junit:junit:4.12"
+            "junit:junit:4.12",
+            "com.github.stefanbirkner:system-rules:1.16.1"
```

Comment:
This dependency is use [`Common Public License 1.0 `](https://github.com/stefanbirkner/system-rules/blob/master/LICENSE), I think we can use here, but this need PPMC to confirm.
By the way, I don't see this jar used at your pr, if it is not used, I don't recommend adding this dependency.

Reply:
It is used to simulation docker ip in env variable .

Reply:
btw, I think it would be ok to use this library. https://github.com/search?q=org%3Aapache+com.github.stefanbirkner&type=code
It is widely used in other apache project now.

Reply:
Cool